guides/doc-Release_Notes/topics/foreman-3.6.2.adoc

```diff
@@ -0,0 +1,37 @@
+= Foreman 3.6.2
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1703[Redmine]
+
+== Foreman
+
+* Pin minitest &lt; 5.19 to resolve test failures - https://projects.theforeman.org/issues/36617[#36617]
+
+=== Host registration
+
+* global registration should not create hosts as "managed" or "to be built" - https://projects.theforeman.org/issues/36393[#36393]
+
+=== Inventory
+
+* Redirect when editing a host is not reliable - https://projects.theforeman.org/issues/36265[#36265]
+* hosts names are hidden in small screens  - https://projects.theforeman.org/issues/36263[#36263]
+* Host Detail button landed to old Host UI page - https://projects.theforeman.org/issues/36225[#36225]
+
+=== Packaging
+
+* Pin sass version to 1.60.z to avoid node 14 dependency - https://projects.theforeman.org/issues/36305[#36305]
+
+=== Unattended installations
+
+* include installation parameter to support puppet 8 - https://projects.theforeman.org/issues/36360[#36360]
```

Comment:
This is an odd one: the issue is still in `New` so the [issues script](https://github.com/theforeman/theforeman-rel-eng/blob/master/issues) should have flagged this as present in Redmine, but not in git. I've now removed the Fixed in releases so it shouldn't show up anymore.

Reply:
So this should be removed?

Reply:
If you regenerate the file it should be gone.